crates/valence_core/src/text.rs

```diff
@@ -555,6 +555,65 @@ impl Text {
             TextContent::StorageNbt { nbt, .. } => nbt.is_empty(),
         }
     }
+
+    /// Attempts to convert the [`Text`] object to a plain string with the [legacy formatting (`§` and format codes)](https://wiki.vg/Chat#Old_system)
+    ///
+    /// Fails if the [`Text`] object contains anything other than text, color
+    /// and styles, or if any of the colors are not on [the legacy color list](https://wiki.vg/Chat#Colors).
+    pub fn try_into_legacy(&self) -> Option<String> {
```

Comment:
"into" usually implies the argument is consumed. Perhaps `try_to_legacy` or `to_legacy`?